dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessService.java

```diff
@@ -686,11 +691,6 @@ private ProcessInstance constructProcessInstance(Command command, String host) {
             } else {
                 processInstance = this.findProcessInstanceDetailById(processInstanceId);
                 // Recalculate global parameters after rerun.
-                processInstance.setGlobalParams(ParameterUtils.curingGlobalParams(
-                    processDefinition.getGlobalParamMap(),
-                    processDefinition.getGlobalParamList(),
-                    getCommandTypeIfComplement(processInstance, command),
-                    processInstance.getScheduleTime()));
```

Comment:
Why is it deleted here?

Reply:
we think the process globalParams should be reset if the commandType is in 'START_PROCESS','SCHEDULER','REPEAT_RUNNING'.those commandType will reset its globalParams in the part of 'switch'
The process globalParams should Keep the original value if the commandType in 'START_CURRENT_TASK_PROCESS' ,'START_FAILURE_TASK_PROCESS' ,and other commandType like the process does not start from the start node,so the globalParams shouldn't be reset.

Reply:
I don't see different processing according to the commandtype